<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetDashboard</class>
 <widget class="QWidget" name="WidgetDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayoutStatus">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Navigator 2 Go Status</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="QLabel" name="labelStatusApache">
          <property name="styleSheet">
           <string notr="true">color: rgb(239, 41, 41)</string>
          </property>
          <property name="text">
           <string>Stopped</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButtonStopApache">
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/stop.png</normaloff>:/icons/icons/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButtonStopWebServer">
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/stop.png</normaloff>:/icons/icons/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://localhost:5000&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;http://localhost:5000&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Remote Uplink:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButtonStartApache">
          <property name="text">
           <string>Start</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/light.png</normaloff>:/icons/icons/light.png</iconset>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="labelDatasetTarget">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Current Dataset Target:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonStartWebServer">
          <property name="text">
           <string>Start</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/light.png</normaloff>:/icons/icons/light.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Web Server (GUnicorn):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelStatusWebServer">
          <property name="styleSheet">
           <string notr="true">color: rgb(239, 41, 41);</string>
          </property>
          <property name="text">
           <string>Stopped</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>THREDDS Server (Apache Tomcat):</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="labelRemoteUplink">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://localhost:8080/thredds&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#007af4;&quot;&gt;http://localhost:8080/thredds&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="quickTools">
     <property name="title">
      <string>Quick Tools</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_13">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Import a *.nc file into the local THREDDS server.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Import NetCDF File(s):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButtonImportNetCDF">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Check Remote Uplink:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButtonCheckRemoteUplink">
        <property name="text">
         <string>Test</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>My Jobs:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QListWidget" name="listWidgetJobs">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Console</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Console output...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditCommandPrompt">
          <property name="placeholderText">
           <string>Linux command prompt</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBoxHDD">
     <property name="title">
      <string>HDD Info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
